scripts/parse_led.py

```diff
@@ -49,7 +49,7 @@
         ofile.write("Please do NOT Edit !!! */\n\n")
 
         ofile.write("static const std::unordered_map<std::string,")
-        ofile.write("std::set<phosphor::led::Layout::LedAction>>")
+        ofile.write(" std::set<phosphor::led::Layout::LedAction>>")
```

Comment:
the change needed here is to replace single quotes with double quotes. But double quotes exists already. so no need to create a 'space' as a change in line 52. you can remove the space.

Reply:
I was getting a conflict here, and I was not sure this space is intentional or by mistake. so I kept it.
Let me know if there is anything else . Will handle all together .